Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -566,7 +566,9 @@
 
 (with-filter (iokit-registry-entry-class "AppleMobileADBE0")
     (deny iokit-get-properties (with no-report)
-        (iokit-property "APTDevice")))
+        (iokit-property
+            "APTDevice"
+            "GainMapVersion")))
```

Comment:
I think this is wrong -- I believe we need GainMapVersion property reading to be supported.

Reply:
Ah, good point, thanks for catching! I have addressed this in the latest patch.

Thanks for reviewing!